src/main/java/Duke.java

```diff
@@ -1,10 +1,294 @@
+import task.*;
+import storage.Storage;
+import ui.Ui;
+
+import java.util.*;
+
 public class Duke {
+
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+
+    public Duke() {
+
+        this.ui = new Ui();
+        this.storage = new Storage(System.getProperty("user.dir") + "/data/", "duke.txt");
+        this.taskList = storage.load();
+
```

Comment:
Should the extra lines between the statements and the method braces exist? It might make the code a little harder to read.

Reply:
In my opinion, its needed because I am initializing new objects.